_posts/2024-09-15-labels-talk/labels-talk.Rmd

```diff
@@ -0,0 +1,158 @@
+We know that putting together a conference submission takes significant effort (especially recording a video) and we sincerely appreciate the time and mental energy that you invested. Thank you again and we hope to still see you at posit::conf(2023).
+
+**My take:** Maybe it isn't exciting? Do I sound like a robot? I state what variable
+labels do but maybe not emphasize enough why you should be using them?
+
+## 2024 -- accepted
+
+**In R:** I had been doing more of the same since my previous submission. But
+perhaps at this time I even more firmly believed not just that creating labelled
+data is is workflow you _could_ incorporate but one that you _should_ incorporate.
+
+**Title:** _Context is King_
+
+**Abstract:** The quality of data science insights is predicated on the practitioner’s understanding of the data. Data documentation is the key to unlocking this understanding; with minimal effort, this documentation can be natively embedded in R data frames via variable labels. Variable labels seamlessly provide valuable data context that reduces human error, fosters collaboration, and ultimately elevates the overall data analysis experience. As an avid, daily user of variable labels, I am excited to help you discover new workflows to create and leverage variable labels in R!
+
+**Email response:**
+
+> Thank you for your submission to posit::conf(2024).
+We’re excited to let you know that it has been accepted as a regular, 20 minute talk!
+We’ll contact you again, at this same email, in the near future to confirm your interest and get some more information from you. These next communications will also provide additional details, such as how to claim your complimentary registration (including instructions for those who have already registered), and information about the speaker training provided by Articulation, Inc.
+Thanks again for your submission, we’re looking forward to hearing from you!
+
+**My take:** I really tried to focus on the human problems that this workflow
+addresses, and I tried to imbue more energy and enthusiasm in the abstract.
+
+
+## Why keep trying?
+
+Some may wonder why I persistently submitted similar abstracts three times.
+It is because I feel that there is not sufficient mainstream documentation
+to demonstrate workflows with labelled data. I suspect that there is a large audience 
+that could benefit from either (1) workflows starting with already labelled data, or (2)
+workflows that create labelled data. And since not many were talking about it,
+I wanted to share what I could via posit::conf in hopes of reaching large audience. 
+However, if my 2024 submission was not accepted, I doubt I would have tried on the same 
+topic again.
+
+## Reflection
```

Comment:
At this stage I really want to know more about how the talk was received, what questions you got etc. Plus maybe something about the training/preparation. In short, was it all worth it :wink: (I'm sure it was!)

Reply:
k i'll add some more in! I did originally think about putting in talk preparation, but then i thought the post might get too long and boring, but if i don't put it here i'll be unlikely to write a separate post about it so might as well.